libraries/lib-string-utils/3party/fast_float.h

```diff
@@ -112,7 +112,8 @@ from_chars_result from_chars_advanced(const char *first, const char *last,
        || defined(__MINGW64__)                                          \
        || defined(__s390x__)                                            \
        || (defined(__ppc64__) || defined(__PPC64__) || defined(__ppc64le__) || defined(__PPC64LE__)) \
-       || defined(__EMSCRIPTEN__))
+       || defined(__EMSCRIPTEN__)) \
+       || defined(_M_ARM64)
```

Comment:
Looks like this line is redundant, _M_ARM64 is checked on line 111

Reply:
Thanks, missed noticing that. Will fix.